torch/_tensor_docs.py

```diff
@@ -326,24 +309,20 @@ def add_docstr_all(method, docstr):
 
 add_docstr_all('copy_',
                r"""
-copy_(src, non_blocking=False, broadcast=True) -> Tensor
+copy_(src, non_blocking=False) -> Tensor
```

Comment:
Is this included in the breaking change list?

Reply:
I've updated the wiki page: https://github.com/pytorch/pytorch/wiki/Breaking-Changes-from-Variable-and-Tensor-merge